coverlay.el

```diff
@@ -555,38 +554,53 @@
   (tabulated-list-print))
 
 ;;;###autoload
-(define-minor-mode coverlay-mode
+(define-minor-mode coverlay-minor-mode
   "overlays for uncovered lines"
   :lighter " lcov"
-  :global t
   :keymap (let ((map (make-sparse-keymap)))
             (define-key map (kbd "C-c C-l l") 'coverlay-toggle-overlays)
             (define-key map (kbd "C-c C-l f") 'coverlay-load-file)
             (define-key map (kbd "C-c C-l w") 'coverlay-watch-file)
             (define-key map (kbd "C-c C-l g") 'coverlay-reload-file)
             (define-key map (kbd "C-c C-l s") 'coverlay-display-stats)
             map)
-  (coverlay--switch-mode coverlay-mode))
+  (coverlay--switch-mode coverlay-minor-mode))
+
+;;;###autoload
+(defun turn-on-coverlay-mode ()
+  "Turn on `coverlay-mode'."
+  (coverlay-minor-mode 1))
+
+;;;###autoload
+(define-globalized-minor-mode global-coverlay-mode coverlay-minor-mode
+  turn-on-coverlay-mode
+  :group 'coverlay
+  :require 'coverlay)
+
+;;;###autoload
+(make-obsolete 'coverlay-mode #'global-coverlay-mode "2.1.0")
+;;;###autoload
+(make-obsolete-variable 'coverlay-mode-hook 'coverlay-minor-mode-hook "2.1.0")
```

Comment:
Please release that version (or another appropriate version number) after this PR is merged.

Reply:
Seems that this PR is a breaking change so it's better to be a version `3.0.0`. Would you update this PR?